deployment/ccip/changeset/deploy.go

```diff
@@ -393,10 +389,6 @@ func configureChain(
 		if err != nil {
 			return err
 		}
-		if enabled, ok := c.USDCConfig.EnabledChainMap()[chainSel]; ok && enabled {
-			ocrParams.ExecuteOffChainConfig.TokenDataObservers = c.USDCConfig.ToTokenDataObserverConfig()
-		}
-		ocrParams.CommitOffChainConfig.PriceFeedChainSelector = cciptypes.ChainSelector(c.FeedChainSel)
```

Comment:
I think much better to just have the full configuration as input and not mutate the config inside here